qutebrowser/browser/hints.py

```diff
@@ -833,7 +833,7 @@ def filter_hints(self, filterstr):
         for elems in self._context.elems.values():
             try:
                 if (filterstr is None or
-                        str(elems.elem).lower().startswith(filterstr)):
+                        filterstr.lower() in str(elems.elem).lower()):
```

Comment:
You're right, casefold is a better option. Although after using the browser today, it might be better with case sensitive matching, to help narrow down the links more precisely. Let me know what you think.



> On 30 Mar 2015, at 03:21, Florian Bruhin <notifications@github.com> wrote:
> 
> In qutebrowser/browser/hints.py:
> 
> > @@ -833,7 +833,7 @@ def filter_hints(self, filterstr):
> >          for elems in self._context.elems.values():
> >              try:
> >                  if (filterstr is None or
> > -                        str(elems.elem).lower().startswith(filterstr)):
> > +                        filterstr.lower() in str(elems.elem).lower()):
> Good idea! However I suggest using str.casefold instead of str.lower. :)
> 
> —
> Reply to this email directly or view it on GitHub.
>